examples/appstream/user.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: appstream.aws.upbound.io/v1beta1
+kind: User
+metadata:
+  annotations:
+    meta.upbound.io/example-id: appstream/v1beta1/user
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    authenticationType: USERPOOL
+    firstName: FIRST NAME
+    lastName: LAST NAME
+    region: us-east-1
+    userName: user@example.com
```

Comment:
If we apply the suggested normalization configuration change, then we will be missing this field:
```suggestion
```

Reply:
I have removed `userName: user@example.com` from the example.